attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java

```diff
@@ -202,7 +202,7 @@ private HttpUrl.Builder getHttpUrlEventsEndpointBuilder() {
 
     @Nullable
     private String parseAttentiveDomainFromTag(String tag) {
-        Pattern pattern = Pattern.compile("window.__attentive_domain='(.*?).attn.tv'");
+        Pattern pattern = Pattern.compile("='([a-z|0-9|/-]+).attn.tv'");
```

Comment:
Suggestion: You can simplify this because square brackets imply multiple groups, and special characters require no escaping if put at the beginning or end of a bracket group.

For example: `"='([a-z0-9-]+).attn.tv'"`

Reply:
@rsmith-attentive do we still need to add the brackets around the periods?